contracts/contracts/oracle/BaseOETHOracleRouter.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../interfaces/chainlink/AggregatorV3Interface.sol";
+import { OracleRouterBase } from "./OracleRouterBase.sol";
+import { StableMath } from "../utils/StableMath.sol";
+
+// @notice Oracle Router that denominates all prices in ETH for base network
+contract BaseOETHOracleRouter is OracleRouterBase {
+    using StableMath for uint256;
+
+    address public immutable aeroPriceFeed;
+
+    constructor(address _aeroPriceFeed) {
+        aeroPriceFeed = _aeroPriceFeed;
+    }
+
+    /**
+     * @notice Returns the total price in 18 digit units for a given asset.
+     *         This implementation does not (!) do range checks as the
+     *         parent OracleRouter does.
+     * @param asset address of the asset
+     * @return uint256 unit price for 1 asset unit, in 18 decimal fixed
+     */
+    function price(address asset)
```

Comment:
If we inherit from OETHOracleRouter, we might wanna initialize the parent contract by passing the Aura price feed address. Since it is designed to work with Base, it made more sense to inherit from the `OracleRouterBase` rather than `OETHOracleRouter.sol`. let me know wdyt.

Reply:
Ah yes you are right I haven't noticed that. What I'd do is create then another abstract contract called `OracleRouterNoPriceRange` that inherits from `OracleRouterBase` and only implements the `price` the way it is implemented in OETHOracleRouter. 

Then alter the code so that `OETHOracleRouter` and `BaseOETHOracleRouter` both inherit from that contract and not re-implement the `price` function.